<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>342</width>
    <height>205</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>342</width>
    <height>205</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>342</width>
    <height>205</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Cachês da Semana</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>imagens/Logo-JAPIASSÚ-símbolo.ico</normaloff>imagens/Logo-JAPIASSÚ-símbolo.ico</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(30, 30, 30);
color: rgb(255, 255, 255);</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>301</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover {
	background-color: rgb(255, 255, 0);
}
QPushButton {
	background-color: rgb(200, 200, 0);
	color:rgb(0,0,0);
	border-radius: 12px;
}
QPushButton:pressed {
	background-color: rgb(200, 200, 0);
	color:rgb(0,0,0);
	border-radius: 12px;
}</string>
    </property>
    <property name="text">
     <string>SALVAR</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>64</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Esta semana</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>64</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Próxima semana</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>23</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Totais:</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>40</y>
      <width>290</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>342</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QMenuBar {
	color: rgb(0,0,0);
	selection-color: rgb(200,200, 0);
	background-color: rgb(200, 200, 0);
	font: 10pt &quot;MS Shell Dlg 2&quot;;
}
QMenuBar::item:selected {
    background: rgb(200, 200,0);
}

QMenuBar::hover {
	color: rgb(50,50,50);
	background-color: rgb(255, 255, 0);
}
 </string>
   </property>
   <widget class="QMenu" name="menuExit">
    <property name="styleSheet">
     <string notr="true">QMenu {
	color: rgb(0,0,0);	
	selection-color: rgb(0, 0, 0);
	selection-background-color: rgb(255, 255, 0);
	background-color: rgb(200, 200, 0);
	font: 9pt &quot;MS Shell Dlg 2&quot;;
}
QMenu::hover {
	color: rgb(30,30,30);
	background-color: rgb(255, 255, 0);
}</string>
    </property>
    <property name="title">
     <string>Arquivo</string>
    </property>
    <addaction name="actionFonte_de_Dados"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuExit"/>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionFonte_de_Dados">
   <property name="text">
    <string>Fonte de Dados</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
